spyder/widgets/about.py

```diff
@@ -161,12 +164,11 @@ def __init__(self, parent):
         btmhlayout.addStretch()
         btmhlayout.addWidget(bbox)
 
-        vlayout = QVBoxLayout()
+        vlayout = QVBoxLayout(self)
         vlayout.addLayout(tophlayout)
+        vlayout.addSpacing(25)
         vlayout.addLayout(btmhlayout)
-
-        self.setLayout(vlayout)
-        self.setFixedSize(410, 560)
```

Comment:
I like that our current About dialog has more height than width. Any chance to get that also in this PR?

Reply:
Maybe we can hardcode a value for the width but let the height free, so that there is not so much empty space between the text and the buttons?

Reply:
For example, if we add `self.label.setFixedWidth(350)`, then it looks like this on my system:

![image](https://user-images.githubusercontent.com/10170372/97060609-cec59b80-1561-11eb-997b-bc18b99a1d3c.png)

Reply:
> Maybe we can hardcode a value for the width but let the height free, so that there is not so much empty space between the text and the buttons?

I like that suggestion. Please go ahead with it.